### PR TITLE
feat: author profile page

### PR DESCRIPTION
**Pull request by fampinheiro**
- [x] Style according to reference page
- [ ] ~~Issue related to dynamic height of image~~ (needs to be solved on card)
- [x] Warning related to prop types

### CODE REVIEW

File: packages/image/image.js [hunk 1]
```diff
@@ -14,6 +14,10 @@ class ImageComponent extends React.Component {
       height: 1
     };
 
+    if (props.source.uri.indexOf("//") === 0) {
+      this.state.source.uri = `https:${  props.source.uri}`;
+    }
+
```
Comment (trevorah) [inline packages/image/image.js]: It shouldnt be the responsibility of this component to clean up dodgy input.

**Comment by trevorah** [comment]
@fampinheiro could you make the footer less shouty? FOOTER!

screen shot 2017-07-27 at 14 34 54

**Comment by fampinheiro** [comment]
I changed the footer to be a black (#1a1f24) box.

image

@trevorah any suggestion?

**Comment by trevorah** [comment]
It'll do for now. Lets get it merged so that we can play with it on some devices.

**Pull request merged**